spec/rubocop/options_spec.rb

```diff
@@ -355,15 +355,6 @@ def abs(path)
       end
     end
 
-    describe '--display-only-fail-level-offenses' do
-      it 'fails if given with an autocorrect argument' do
-        %w[--fix-layout -x --autocorrect -a --autocorrect-all -A].each do |o|
-          expect { options.parse ['--display-only-correctable', o] }
-            .to raise_error(RuboCop::OptionArgumentError)
```

Comment:
It seems that instead of being removed, it should be updated to reflect an expected behavior, rather than resulting in the `RuboCop::OptionArgumentError` error.

Reply:
I'm not sure I follow. This tests the validation that I removed in order to allow the two arguments to be used together. I could change it to `not_to` expect the error, but did you have something else in mind?

Reply:
Done. The test was actually not doing the right thing before, because it was re-using the same `Options` instance and generating errors for a different reason (using -a and -A together)